packages/tokens/non-fungible/src/extensions/royalties/mod.rs

```diff
@@ -0,0 +1,99 @@
+mod storage;
+use crate::NonFungibleToken;
+
+mod test;
+
+use soroban_sdk::{Address, Env};
+
+/// Royalties Trait for Non-Fungible Token (ERC2981)
+///
+/// The `NonFungibleRoyalties` trait extends the `NonFungibleToken` trait to
+/// provide the capability to set and query royalty information for tokens. This
+/// trait is designed to be used in conjunction with the `NonFungibleToken`
+/// trait.
+///
+/// This implementation follows the ERC2981 standard for royalties, allowing:
+/// - Setting global royalties for the entire collection
+/// - Setting per-token royalties that override the global setting
+/// - Enforcing a maximum royalty percentage to prevent scams
```

Comment:
Is this still the case?

Reply:
Thanks! And let's not resolve this convo as a reminder, maybe the discussion with Jane result in another change 👍